tests/approvals/arc4-types.algo.ts

```diff
@@ -160,8 +163,25 @@ export class Arc4TypesTestContract extends BaseContract {
     testUFixed()
     testDynamicBytes(Bytes('hmmmmmmmmm'))
     testStaticBytes()
+    testZeroValues()
     const result = new arc4.DynamicArray<arc4.UintN<64>>()
     assert(result.length === 0)
     return true
   }
 }
+
+function testZeroValues() {
+  assert(new StaticArray<UintN8, 4>().bytes === bzero(4))
+  assert(new StaticArray<Bool, 4>().bytes === bzero(1))
+  assert(new StaticArray<Bool, 9>().bytes === bzero(2))
+  assert(new StaticArray<Str, 4>().bytes === bzero(4 * 2))
+  assert(new Tuple<[UintN8, Bool, Bool, Str]>().bytes === bzero(4))
```

Comment:
This is an example where the result shouldn't be all zeros due to the `Str` element.